jest.config.js

```diff
@@ -0,0 +1,23 @@
+/* global module */
+const { jestConfig } = require("@salesforce/sfdx-lwc-jest/config");
+
+const setupFilesAfterEnv = jestConfig.setupFilesAfterEnv || [];
+setupFilesAfterEnv.push("<rootDir>/jest.setup.js");
+
+// Require 100% LWC code coverage
+const coverageThreshold = jestConfig.coverageThreshold || {};
+coverageThreshold.global = {
+    branches: 50,
+    functions: 100,
+    lines: 50,
+    statements: 0,
```

Comment:
@jstvz the jest docs are a little unclear here, but it appears that `branches`, `functions`, and `lines` are a percentage while statements are a number:

> For example, with the following configuration jest will fail if there is less than 80% branch, line, and function coverage, or if there are more than 10 uncovered statements:
```
{
  ...
  "jest": {
    "coverageThreshold": {
      "global": {
        "branches": 80,
        "functions": 80,
        "lines": 80,
        "statements": -10
      }
    }
  }
}
```

Reply:
In your example it is counting statements since it's a negative number. Changing your example to add `"lines": -80,` would change the behavior from "≥80% of lines" to "no more than 80 uncovered lines".

Agreed, docs are rather muddy.

Reply:
Confirming:
```diff
diff --git a/force-app/main/default/lwc/dataTableWrapper/dataTableWrapper.js b/force-app/main/default/lwc/dataTableWrapper/dataTableWrapper.js
index 9062ab6..68434c2 100644
--- a/force-app/main/default/lwc/dataTableWrapper/dataTableWrapper.js
+++ b/force-app/main/default/lwc/dataTableWrapper/dataTableWrapper.js
@@ -1,5 +1,10 @@
 import { LightningElement, api } from "lwc";
 
+const foo = () => {
+    console.log("This is uncovered");
+    console.log("This is also uncovered");
+}
+
 export default class DataTableWrapper extends LightningElement {
     @api keyField = "id";
     @api records = [];
diff --git a/jest.config.js b/jest.config.js
index 7b5b1d5..c7de1c4 100644
--- a/jest.config.js
+++ b/jest.config.js
@@ -8,9 +8,9 @@ setupFilesAfterEnv.push("<rootDir>/jest.setup.js");
 const coverageThreshold = jestConfig.coverageThreshold || {};
 coverageThreshold.global = {
     branches: 50,
-    functions: 100,
-    lines: 50,
-    statements: 0,
+    functions: 50,
+    lines: -1,
+    statements: 100,
 };
 
 module.exports = {
```

Test result:
```console
$ npx lwc-jest --coverage -- --passWithNoTests
 PASS  force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js
  c/dataTableWrapper
    ✓ FIXME should get code coverage for handleRowChange (17ms)
    ✓ should be accessible (67ms)

---------------------|---------|----------|---------|---------|-------------------
File                 | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
---------------------|---------|----------|---------|---------|-------------------
All files            |   77.78 |      100 |   66.67 |      75 |                   
 dataTableWrapper.js |   77.78 |      100 |   66.67 |      75 | 4-5               
---------------------|---------|----------|---------|---------|-------------------
Jest: "global" coverage threshold for statements (100%) not met: 77.78%
Jest: Uncovered count for lines (2)exceeds global threshold (1)
Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        2.177s
Ran all test suites.

$ echo $?
1
```